gulpfile.js

```diff
@@ -27,6 +27,7 @@ gulp.task('build', function build() {
 gulp.task('fetch', async () => {
   const { sdkKey } = argv;
   if (!sdkKey) {
+    // eslint-disable-next-line no-console
```

Comment:
I'd suggest injecting as little information into the webpack build as possible.

That looks to be the environment key. I'd also suggest using a format like we do in other builds to indicate defines, such as:

`__SDK_KEY__`